footil/formatting.py

```diff
@@ -285,6 +285,19 @@ def replace_email_name(name: str, oldemail: str) -> str:
     return formataddr((name, email_part))
 
 
+def replace_email(email_part: str, oldemail: str) -> str:
```

Comment:
Maybe, `orig/original_email` would fit better here?
```suggestion
def replace_email(email_part: str, old_email: str) -> str:
```

Reply:
Then need to update replace_email_name